include/mbedtls/fsio.h

```diff
@@ -0,0 +1,255 @@
+/**
+ * \file fsio.h
+ *
+ * \brief File read/write functions
+ *
+ *  Copyright (C) 2017, ARM Limited, All Rights Reserved
```

Comment:
A sad reminder of how long this PR is open: Could you update the year?

Reply:
Just in anticipation can I make it 2019  😉